kotlinx-coroutines-debug/README.md

```diff
@@ -154,9 +154,8 @@ of the dump programmatically.
 
 ### Debug agent and Android
 
-Unfortunately, Android runtime does not support Instrument API necessary for `kotlinx-coroutines-debug` to function, triggering `java.lang.NoClassDefFoundError: Failed resolution of: Ljava/lang/management/ManagementFactory;`.
-
-Nevertheless, it will be possible to support debug agent on Android as soon as [GradleAspectJ-Android](https://github.com/Archinamon/android-gradle-aspectj)  will support android-gradle 3.3 
+Android runtime does not support Instrument API necessary for `kotlinx-coroutines-debug` to function, triggering `java.lang.NoClassDefFoundError: Failed resolution of: Ljava/lang/management/ManagementFactory;`,
+and it is not possible to use coroutine debugger along with Android emulator.
```

Comment:
Are there any public statements about Android Studio eventually supporting this? If so, maybe we should list them here?

Reply:
No, there aren't. All we have is the allowance to refer to the "AS will support coroutines debugger eventually" (probably in the next major release, AFAIR most of the work on AS and our side is done)